introducing-java/src/test/java/com/onto/javacourse/intro/gcd/GCDTest.java

```diff
@@ -0,0 +1,20 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GCDTest {
```

Comment:
This is good but you have an if statement in both methods. Can you also test the exception throwing, see https://www.baeldung.com/junit-assert-exception